test/markdownlint-test.js

```diff
@@ -1138,9 +1138,13 @@ module.exports.readme = function readme(test) {
             if (rule) {
               const ruleName = rule.names[0];
               const ruleAliases = rule.names.slice(1);
-              const expected = "**[" + ruleName + "](doc/Rules.md#" +
+              let expected = "**[" + ruleName + "](doc/Rules.md#" +
                 ruleName.toLowerCase() + ")** *" +
                 ruleAliases.join("/") + "* - " + rule.description;
+              const expectedIfDeprecated = "~~" + expected + "~~";
+              if (token.content === expectedIfDeprecated) {
```

Comment:
This is clever, but could you please do something like: https://github.com/DavidAnson/markdownlint/blob/1ae512be697e56de7d397e93b8f8e01b949da147/lib/markdownlint.js#L13

It is a bit more strict and I am paranoid. :)

Reply:
Same comment below.

Reply:
Fixed